test/JavaDataCollectorTest.java

```diff
@@ -7,38 +8,59 @@
 import services.dataAccess.InMemoryAccessObject;
 import services.dataAccess.proto.PostProto.Post;
 import services.sources.TwitterSource;
+import services.sources.RedditSource;
 
 import java.util.List;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 
 public class JavaDataCollectorTest {
 
 	AbstractDataAccess data;
-	TwitterSource source;
-	AbstractDataCollector collector;
+	TwitterSource twitterSource;
+	RedditSource redditSource;
+	AbstractDataCollector twitterCollector;
+	AbstractDataCollector redditCollector;
```

Comment:
This is fine for now because we don't have something set up to do mocks, but in the future we should have a mock collector so that we are only testing JavaDataCollector in this class and not the collectors it uses.